java/sage/miniclient/MiniMPlayerPlugin.java

```diff
@@ -544,7 +561,16 @@ else if (hostname != null && hostname.length() > 0)
         else
           cmds.add(new java.io.File(file).getPath());
 
-        if ("TRUE".equals(MiniClient.myProperties.getProperty("player_cmdline_debug", null))) System.out.println("Executing mplayer cmd: " + cmds);
+        String args[] = (String[])cmds.toArray(new String[0]);
```

Comment:
Now that you're creating an args[], you can reuse it when launching mplayer below. :)